src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -23,7 +23,7 @@
         "symfony/http-foundation": "~2.7|~3.0",
         "symfony/http-kernel": "~2.7|~3.0",
         "symfony/filesystem": "~2.7|~3.0",
-        "symfony/routing": "~2.7|~3.0",
+        "symfony/routing": "~3.0",
```

Comment:
No it's not unfortunately. You should adjust the tests to make them less strict. With [assertStringMatchesFormat](https://phpunit.de/manual/current/en/appendixes.assertions.html#appendixes.assertions.assertStringMatchesFormat) it should be possible.